src/s390x/zipl.rs

```diff
@@ -150,20 +150,29 @@ fn generate_sdboot(
         .write_all(options.as_bytes())
         .context("writing zipl se cmdline")?;
 
-    let mut lukskeys_path = "/etc/luks/".to_string();
-    let mut crypttab_path = "/etc/crypttab".to_string();
-    let mut hostkeys_path = "/etc/se-hostkeys/".to_string();
-
     // during cosa-build rootfs includes ostree commit path
-    if let Some(ref rootfs) = rootfs {
-        lukskeys_path.insert_str(0, rootfs);
-        crypttab_path.insert_str(0, rootfs);
-        hostkeys_path.insert_str(0, rootfs);
-    }
+    let ostree_commit = if let Some(ref rootfs) = rootfs {
```

Comment:
Aside: can't we drop the `--rootfs` arg from the `rdcore zipl` command now? I think all instances where this code is called, it's done from within the deployment root.

But maybe better to keep it for now to make the switch to the new dm-verity approach easier.

Reply:
i'd wait with dropping til dm-verity replaces LUKS  in both FCOS & RHCOS